lib/analyzer.js

```diff
@@ -330,11 +341,14 @@ Analyzer.prototype._dependenciesLoaded = function _dependenciesLoaded(href, root
  * @param  {string} href     The href to get dependencies for.
  * @return {Array.<string>}  A list of all the html dependencies.
  */
-Analyzer.prototype._dependencies = function _dependencies(href, found) {
+Analyzer.prototype._dependencies = function _dependencies(href, found, transitive) {
```

Comment:
naming nit: `_dependencies` should be a getter or if left as a method, named as a verb.